mobile/src/home-view.js

```diff
@@ -123,15 +123,15 @@ class HomeView extends PureComponent {
         })
         fbc.database.public
         .adminRef('questionPrompt')
-        .on('value', data => this.setState({ questionPrompt: data.val() || '' }))
+        .on('value', data => this.setState({ questionPrompt: data.val().trim() || '' }))
```

Comment:
if `data.val()` is null (no data) this will crash.  Instead trim `data.val() || ''`